test/integration/discovery.v2.test.js

```diff
@@ -38,7 +37,181 @@ describe('discovery v2 integration @slow', () => {
 
   let documentId;
 
+  const EXTENDED_TIMEOUT = 30000;
+
+  describe('projects', () => {
+    test(
+      'projects CRUD',
```

Comment:
I see where you're coming from here, but I think we should hold to the testing principle that each test should cover functionality for one individual method. It is generally not good practice to include too much functionality in one test. If one of these requests were to fail, it would not be trivial to know which one - the failing test would just be `projects CRUD` and we would have to look at the line number in the code to see which operation failed.

Also, I should have mentioned this earlier but since you're writing new tests for a bunch of operations, we should clean up the format a little. The try/catch thing actually has some hidden potential for issues and it is not necessary. I updated the code in our Node template and you should probably match that format for the new tests (not required to backfit these changes to all tests at this point). [Here is a super bare-bones example](https://github.ibm.com/CloudEngineering/node-sdk-template/blob/master/test/integration/example-service.v1.test.js#L84-L90)

Reply:
I was honestly going based off Jeff's suggestion on how to make CRUD tests but I can certainly separate them 😅

Reply:
Yeah I can see an argument for doing it that way, because I don't think tests are guaranteed to run in a certain order, but they always have run in the order we want them to so I think it's worth the risk to make the tests more granular.